updater/aws.go

```diff
@@ -60,6 +62,37 @@ type EC2API interface {
 	WaitUntilInstanceStatusOk(input *ec2.DescribeInstanceStatusInput) error
 }
 
+func (u *updater) anotherUpdaterRunning(family string) (bool, error) {
```

Comment:
nit: I think a better name for this may be `alreadyRunning` or `isOnlyUpdaterRunning`

Reply:
adopted `alreadyRunning` name.